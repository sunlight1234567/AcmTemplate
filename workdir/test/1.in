5
0 1
10 10
10 11
101 101
110 110
