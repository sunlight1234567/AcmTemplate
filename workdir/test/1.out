0
100
110
1
10110100
